salt/states/file.py

```diff
@@ -1832,14 +1835,7 @@ def symlink(
                         name, backupname, exc
                     ),
                 )
-        elif force:
```

Comment:
same question here. Why are you removing the logic to force the removal of link if force is set.

Reply:
that logic is handled in the execution module. note that I'm passing `force` to it, whereas it wasn't there before